README.md

```diff
@@ -75,32 +75,41 @@ Once all the required packages are installed, enter the virtual environment with
 
     pipenv shell
 
-The following runtime flags are available:
+To then run the app, use the command:
+
+    flask run
+
+This will cause the crawler to execute an ingest every 15 minutes, triggered by cron, so at 0, 15, 30 and 45 minutes past the hour.
+This scheduled behaviour can be turned off by adding the following to the `development.py` file:
+
+    SCHEDULER_RUN = False
+
+You can also adjust the behaviour of the scheduled ingest using the settings in the same file.
+To run an ingest immediately, whether Flask is running or not, the `runner.py` file can be used with the arguments shown:
 
     python runner.py --help
 
-    usage: runner.py [-h] [--sftp] [--scheduled]
+    usage: runner.py [-h] [--sftp] [--keep-files] [--add-to-dart] [--centre_prefix {ALDP}]
```

Comment:
After merging rom develop, the `--centre_prefix` is now a list of all the centres.